pkg/apis/agones/v1/gameserver.go

```diff
@@ -135,16 +135,20 @@ type GameServerSpec struct {
 	Health Health `json:"health,omitempty"`
 	// Scheduling strategy. Defaults to "Packed"
 	Scheduling apis.SchedulingStrategy `json:"scheduling,omitempty"`
-	// Logging specifies log levels for Agones system containers
-	Logging Logging `json:"logging,omitempty"`
+	// SdkServer specifies parameters for the Agones SDK Server sidecar container
+	SdkServer SdkServer `json:"sdkServer,omitempty"`
 	// Template describes the Pod that will be created for the GameServer
 	Template corev1.PodTemplateSpec `json:"template"`
 }
 
-// Logging specifies log levels for Agones system containers
-type Logging struct {
-	// SdkServer is a log level for SDK server (sidecar) logs. Defaults to "Info"
-	SdkServer string `json:"sdkServer,omitempty"`
+// SdkServer specifies parameters for the Agones SDK Server sidecar container
+type SdkServer struct {
+	// Logging is a log level for SDK server (sidecar) logs. Defaults to "Info"
+	Logging string `json:"logging,omitempty"`
```

Comment:
Probably not for this PR, but should this be its own type, with predefined values as variables - LoggingError, LoggingWarning, and LoggingInfo ?

Reply:
Maybe? Are there examples of other enums being done this way instead of just using a string with documentation?

Reply:
Yeah, lots, e.g.:
https://github.com/googleforgames/agones/blob/master/pkg/apis/agones/v1/gameserver.go#L36-L71

Pattern taken from Kubernetes e.g.:
https://github.com/googleforgames/agones/blob/master//vendor/k8s.io/api/core/v1/types.go#L345-L357 (and lots more in that file)

/cc @aLekSer for visibility. I should have picked this up on initial review, apologies.

Reply:
I've gone ahead and done this in the same PR. No need to create another one for this change.